manager-bundle/composer.json

```diff
@@ -46,7 +46,8 @@
         "symfony/twig-bundle": "4.2.* || 4.3.*",
         "symfony/web-profiler-bundle": "4.2.* || 4.3.*",
         "symfony/yaml": "4.2.* || 4.3.*",
-        "toflar/psr6-symfony-http-cache-store": "^2.0"
+        "toflar/psr6-symfony-http-cache-store": "^2.0",
```

Comment:
A new dependency just to check the file extension?🤔

Reply:
We already require `webmozart/path-util` in the core bundle, but I you certainly have a point here. Maybe we should just use `pathinfo()`, which the method does internally as well.